server/models/README.md

```diff
@@ -0,0 +1,55 @@
+Bhima Models
+============
+
+This document describes the contents of the `/models` directory.
+
+The directory contents are listed below:
+```bash
```

Comment:
It renders it nice and pretty in github-flavored markdown.  I guess it's probably going to be read through a text editor, so I'll take it out.